Force pytraj to regenerate its Cython-generated extension sources.
The pre-generated C/C++ sources included with AmberTools 26 are not
compatible with Python 3.14. Disable use of the bundled pre-generated
sources so that pytraj regenerates them with the Cython version available
in the build environment.
Author: Pavel Tomanek (Inuits/UGent) with help from ChatGPT5.6
diff --git a/AmberTools/src/pytraj/base_setup/build_config.py b/AmberTools/src/pytraj/base_setup/build_config.py
--- a/AmberTools/src/pytraj/base_setup/build_config.py
+++ b/AmberTools/src/pytraj/base_setup/build_config.py
@@ -17,7 +17,7 @@ class BuildConfig:
         self.disable_openmp = self._check_flag('--disable-openmp')
         self.use_amberlib = not self._check_flag('--disable-amberlib')
         # self.use_prebuilt = self._check_flag('--use-pre-cythonized')
-        self.use_prebuilt = True
+        self.use_prebuilt = False
         self.debug = self._check_flag('-debug')
         self.cythonize_only = self._check_flag('--cythonize')